config/config.go

```diff
@@ -214,6 +214,9 @@ func isExecutable(path string) bool {
 func isInShells(path string) bool {
 	file, err := os.Open("/etc/shells")
 	if err != nil {
+		if path == "/bin/sh" { // if no /etc/shell is found, /bin/sh is accepted
```

Comment:
thank you for the fix, it makes sense. could you use [DefaultShellCommand](https://github.com/mendersoftware/mender-connect/blob/master/config/paths.go#L28) here?

Reply:
sure, will do that

Reply:
done

Reply:
thank you!
I know it is annoying, but now this line is too long, our static checker will not let it in:
```go
config/config.go:217: line is 102 characters (lll)
		if path == DefaultShellCommand { // if no /etc/shell is found, DefaultShellCommand is accepted
```
could we maybe just move the comment up?

```go
		// if no /etc/shell is found, DefaultShellCommand is accepted
		if path == DefaultShellCommand { 
```

Reply:
fixed

Reply:
thank you.